addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -504,6 +508,21 @@ export class SelectionPlugin extends Plugin {
     setCursorEnd(node) {
         return this.setSelection({ anchorNode: node, anchorOffset: nodeSize(node) });
     }
+    /**
+     * Force the cursor after the given node. (normalize : false)
+     * @param { Node } node
+     */
+    forceCursorAfter(node) {
+        const parentElement = node.parentElement;
+        if (parentElement === this.editable && !this.config.allowInlineAtRoot) {
+            return false;
+        }
+        const nodeIndex = Array.from(parentElement.childNodes).indexOf(node);
+        return this.setSelection(
+            { anchorNode: parentElement, anchorOffset: nodeIndex + 1 },
+            { normalize: false }
+        );
+    }
```

Comment:
are you sure you need to share a function for this?
Unless I'm missing something about `allowInlineAtRoot`, calling this function seems equivalent to
```js
const [anchorNode, anchorOffset] = rightPos(node);
this.dependencies.selection.setSelection({ anchorNode, anchorOffset }, { normalize: false });
```